addons/binding/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MystromDeviceAddedListener.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
```

Comment:
The year 2014 should be 2010. You also might run mvn install in your binding and look at the `target/code-analysis/report.html` to look for coding style issues.